.github/workflows/main.yml

```
@@ -1,61 +1,50 @@
-          make test
+      - name: Build Docker image
+        uses: docker/build-push-action@v4
+        with:
+          push: false
+          tags: ${{ env.BUILD_ENGINE_BUILD_TAG }}
+          context: .
+          load: true
 
-      - name: Build docker image
+      - name: Build OTEL Docker image
         uses: docker/build-push-action@v4
```

actionlint (1.7.11)

[error] the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)